src/client/App.jsx

```diff
@@ -1,77 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { renderRoutes } from 'react-router-config';
 
-import { getMoviesFromLocalStorage, randomSearch, addNotification } from './actions/actions';
+import { randomSearch } from './actions/actions';
 
 // Page components
```

Comment:
@vitkuz this comment is not needed, right?

Reply:
Done